lesson_02/index.ko.md

```diff
@@ -0,0 +1,181 @@
+# FFmpeg 어셈블리어 강의 2강
+
+첫 번째 어셈블리어 함수를 작성해 보았습니다. 이번에는 분기(branch)와 반복문(loop)을 소개하겠습니다.
+
+먼저 레이블과 점프(jump)의 개념을 알아야 합니다. 아래의 예제에서 `jmp` 명령어는 코드 실행을 `.loop:` 뒤로 이동시킵니다. `.loop:`는 **레이블**이라 부르며, 레이블 앞에 붙은 점(`.`)은 이것이 로컬 레이블(local label)임을 나타냅니다. 로컬 레이블을 사용하면 여러 함수에서 동일한 레이블을 재사용할 수 있습니다.
```

Comment:
This single paragraph is already tough! Here are some suggestions

- "`.loop:` 뒤로 이동시킵니다": to specify the next instruction, I suggest ".loop:` 직후의 명령으로 이동시킵니다".
- 로컬 레이블: Instead of transliteration of 'local' to '로컬', how about '지역'? There are a number of examples for this direction such as 지역 변수 (local variable).